userdetails-cognito/grails-app/conf/application.yml

```diff
@@ -158,6 +158,8 @@ openapi:
         scopes:
           users/read: Read or list any or all user records
           users/write: Write to a user's record, eg update a user's attributes.
+  server-urls:
+    - https://api.ala.org.au/userdetails
```

Comment:
This shouldn't be hardcoded here, it should be in the external config in the ansible inventory IMHO.

Reply:
Fixed.